directorynameproject2/directorynameproject2/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Questionaire
+{
+   public class Program
+    {
+        public static decimal GetDecimalAnswerFromUser(string question) //this method will be called a bunch below
+        {
+            Console.WriteLine(question);
+            string response = Console.ReadLine();
+            decimal answer = decimal.Parse(response);
+            return answer;
+        }
+        public static void Main(string[] args)
+        {
+            Questionaire display = new Questionaire(); //constructor
```

Comment:
I don't think that "display" makes sense as a variable name here. What is a display?  What does it mean for a display to have a number of pets?  Perhaps "user" or something similar would be better